docs/api/index.rst

```diff
@@ -20,5 +20,40 @@ Producttype API         `0.0.2 <https://redocly.github.io/redoc/?url=https://raw
 In addition, Open Product can work with `Notificaties API`_. Open Product uses
 `Open Notificaties`_ by default but this can be disabled using ``NOTIFICATIONS_DISABLED`` (see :ref:`installation_env_config`).
 
+The Open Product API is compliant with most of the `NLGov REST API Design Rules`_:
+
+Functional rules
+----------------
+
+- `core/naming-resources <https://gitdocumentatie.logius.nl/publicatie/api/adr/2.0.2/#/core/naming-resources>`_
```

Comment:
To make it look a bit better, you could put this in a table (similar to https://objects-and-objecttypes-api.readthedocs.io/en/latest/api/compliancy/vng.html), but it's a minor thing